helpers/dependencies/docker-compose.js

```diff
@@ -2,7 +2,7 @@ const { spawn } = require('child_process');
 
 const dockerCompose = async () =>
 	new Promise((resolve, reject) => {
-		const _spawn = spawn('docker-compose', ['-v']);
+		const _spawn = spawn('docker-compose', ['-v'], { shell: true });
```

Comment:
@syedyaseenabbas please change `docker-compose` to `docker compose`

Reply:
You'd need to move "compose" in array of string of 2nd param of spawn.

Reply:
Done.

Reply:
@syedyaseenabbas I don't see the changes, check if changes were pushed.